tests/test_views_collections.py

```diff
@@ -52,7 +52,9 @@ def test_collections_are_isolated_by_bucket(self):
         self.app.put_json('/buckets/sodas',
                           MINIMALIST_BUCKET,
                           headers=self.headers)
-        self.app.get(other_bucket, headers=self.headers, status=404)
+        resp = self.app.get(other_bucket, headers=self.headers, status=404)
+        self.assertIn('id', resp.json['details'])
+        self.assertIn('resource_name', resp.json['details'])
```

Comment:
Usually we only keep the assertions that are relevant to the specification (ie. the test title).

For example, the test on the values of id and details is not properly relevant for the spec «*collections are isolated by bucket*».

I suggest that you only keep the assertions regarding `details` in dedicated tests similar to `test_unknown_collection_raises_404` from `test_views_records.py`, but for each of bucket, group, collection and record in the other `test_views_` files.